src/auxiliary/math.jl

```diff
@@ -23,44 +23,46 @@ function set_polyester!(toggle::Bool; force = true)
 end
 
 """
-    Trixi.set_sqrt_type(type; force = true)
+    Trixi.set_sqrt_type!(type; force = true)
 
 Set the `type` of the square root function to be used in Trixi.jl.
 The default is `"sqrt_Trixi_NaN"` which returns `NaN` for negative arguments
 instead of throwing an error.
-Alternatively, you can set `type` to `"sqrt_Base"` to use the Julia built-in `sqrt` function 
+Alternatively, you can set `type` to `"sqrt_Base"` to use the Julia built-in `sqrt` function
 which provides a stack-trace of the error which might come in handy when debugging code.
 """
-function set_sqrt_type(type; force = true)
+function set_sqrt_type!(type; force = true)
     @assert type == "sqrt_Trixi_NaN"||type == "sqrt_Base" "Only allowed `sqrt` function types are `\"sqrt_Trixi_NaN\"` and `\"sqrt_Base\"`"
     set_preferences!(TRIXI_UUID, "sqrt" => type, force = force)
     @info "Please restart Julia and reload Trixi.jl for the `sqrt` computation change to take effect"
 end
 
+@deprecate set_sqrt_type(type; force = true) set_sqrt_type!(type; force = true) false
```

Comment:
```suggestion
# TODO: deprecation introduced in v0.8
@deprecate set_sqrt_type(type; force = true) set_sqrt_type!(type; force = true) false
```

This makes it easier to find deprecation introduced in a specific version when preparing a breaking release (since we could not use `@deprecate` every time).